### DISCUSSION TITLE
Arduino Matter support

### DISCUSSION THREAD

**Discussion by Blaster1920** [Makuna/NeoPixelBus#862] [General]
Hi all,
do you think the new [Arduino Matter](https://store.arduino.cc/pages/nano-matter) could be supported by NeoPixelBus?
I'm looking for a device to integrate strips into Home Assistant. The Matter has a radio module that is Zigbee compatible, so should be spot on.

**Comment by Makuna** [comment]
Newer platforms require a great effort to support.  I try to do more than simple bitbang methods as in the past they prove to problematic when used with RF features like WiFi, Bluetooth, and Zigbee.  This requires really good documentation to be available on some of the more advanced features of the chip, which always seems to be lacking for a new platform.  Being ARM doesn't make this any easier, as it really doesn't prescribe how hardware peripheral features are standardized, so almost every board is unique.  

You are welcome to add an ISSUE to track it though.  Include any links you find, like other libraries that support it for WS2812.
